titus-api/src/main/java/com/netflix/titus/api/jobmanager/JobAttributes.java

```diff
@@ -27,6 +27,11 @@
 
     // Job Descriptor Attributes
 
+    /**
+     * Job creator.
+     */
+    public static final String JOB_ATTRIBUTES_CALLER_ID = TITUS_ATTRIBUTE_PREFIX + "caller.id";
```

Comment:
should the attributed by called `createdBy`?

Reply:
expanding: `caller.id` could mean any operation, not necessarily who submitted the job

Reply:
You are right. I will change this.